docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -257,14 +257,22 @@ This behavior can be disabled by setting `autoRegister=false` in `@EnableDiscove
 NOTE: `@EnableDiscoveryClient` is no longer required.
 You can put a `DiscoveryClient` implementation on the classpath to cause the Spring Boot application to register with the service discovery server.
 
-==== Health Indicator
+==== Health Indicators
 
-Commons creates a Spring Boot `HealthIndicator` that `DiscoveryClient` implementations can participate in by implementing `DiscoveryHealthIndicator`.
-To disable the composite `HealthIndicator`, set `spring.cloud.discovery.client.composite-indicator.enabled=false`.
-A generic `HealthIndicator` based on `DiscoveryClient` is auto-configured (`DiscoveryClientHealthIndicator`).
-To disable it, set `spring.cloud.discovery.client.health-indicator.enabled=false`.
-To disable the description field of the `DiscoveryClientHealthIndicator`, set `spring.cloud.discovery.client.health-indicator.include-description=false`.
+Commons auto-configures the following Spring Boot health indicators.
```

Comment:
@ryanjbaxter please take a scan on these changes. When I went through to add the updated property I saw it was a bit confusing so I went ahead and updated the section to clarify a bit.